pkg/optimizer/optimizer_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package optimizer
+
+import (
+	"os"
+	"testing"
+)
+
+func TestOptimizer(t *testing.T) {
+	env := getTestEnv()
+	opt := NewOptimizer(env, getDataInfo(env), os.Getenv("ABSTOOLBIN")+"/fzn-or-tools")
+	solution, err := opt.Solve()
+	if err != nil {
+		t.Fatalf("Failed solving constraint problem: %s", err)
+	}
+
+	solutionLen := len(solution.DataPath)
+	if solutionLen < 1 {
+		t.Error("Solution is too short")
```

Comment:
Does this mean that the solution does not exist?

Reply:
Fixed

Reply:
Fixed (should have been `solutionLen < 2`)